### DISCUSSION TITLE
I'm looking for more details regarding GitHub Certification exams

### DISCUSSION THREAD

**Discussion by vitalykarasik** [community/community#86093] [GitHub Education] [answered]
I saw https://github.blog/2024-01-08-github-certifications-are-generally-available/#verified-credentials-with-credly announcement and reviewed suggested GitHub Training modules, but I don't see information about exams.
Is it a practical exam, or just questions?

Thanks, Vitaly

**Comment by Shahnawazk7944** [comment]
You're right, the GitHub blog post announcing the general availability of GitHub certifications focuses on the verified credentials aspect and doesn't delve into specifics about the exam format. However, based on the information available on the GitHub Learn website and discussions within the GitHub community, here's what we can gather about the exams:

**Exam Format:**

* **Multiple-choice questions:** The exams seem to primarily consist of multiple-choice questions that test your understanding of the relevant concepts and skills.
* **Scenario-based questions:** Some discussions suggest there might be a few scenario-based questions as well, where you'll be presented with a situation and asked to choose the best course of action.
* **No coding required:** As of now, there's no indication that the exams require you to write actual code. They focus on theoretical knowledge and practical application of concepts within the GitHub platform.

**Exam Difficulty:**

* **Beginner-friendly for Foundations:** The GitHub Foundations exam is designed for individuals new to GitHub and aims to assess their understanding of basic concepts like Git, repositories, collaboration features, etc. So, it's expected to be beginner-friendly.
* **Intermediate for Advanced Security:** The GitHub Advanced Security (GHAS) certification exam delves into more advanced security features and workflows specific to GitHub Enterprise. This one would cater to developers with some experience in security practices.

**Finding More Information:**

* **GitHub Learn website:** Keep an eye on the GitHub Learn website for any updates or announcements regarding exam formats, practice resources, or study guides.
* **GitHub Discussions:** Actively participating in relevant discussions within the GitHub community can provide valuable insights from experienced users and developers who have taken the exams.

Remember, the GitHub certifications are still relatively new, and the platform might be refining the exam formats and resources over time. So, staying updated and actively seeking information is crucial for optimal preparation.

I hope this clarifies the current understanding of the GitHub certification exams based on the available information. Let me know if you have any further questions!

**Reply by vitalykarasik** [reply]
Thank you!
Personally, I won't take these exams if they are theoretical - I like and believe only in practical tasks with open Internet.

**Comment by FidelusAleksander** [comment, marked as answer]
@vitalykarasik 
These exams consist of 75 questions, out of which about 60 are scored.

Some questions are single answer questions and some are multiple answer, but you should be told on how many correct answers there are.

I've created practice exams that follow that same schema.
Repository: https://github.com/FidelusAleksander/githubcertified
Website with practice exams: https://githubcertified.com/

**Reply by vitalykarasik** [reply]
Thank you! As far as I see, your exams are multiple-choice questions as well, i.e. not practical, right?

**Reply by FidelusAleksander** [reply]
@vitalykarasik I'm mirroring the exam experiannce, where some questions are single-choice and some are multiple-choice

**Reply by PRIYABRATA2** [reply]
I have a doubt, when appearing can we give answers to only 60 questions out of 75 , as its getting evaluated out of 60. If we appear all questions , may be initial 60 questions will get considered. Any thoughts ?
